core/commands/generate_repository_partials/repository_parser_core.rb

```diff
@@ -522,15 +522,15 @@ def add_unsupported_repo(main_path, unsupported_path, auth, log, logger)
   end
 
   PLATFORMS = {
-    'jammy' => {
+    'ubuntu22.04' => {
```

Comment:
Is this change safe for other parsers?

Reply:
S3 parser is the only one that utilizes hash names by using distro/version name from the repo link as key. Other functions loop through the `PLATFORMS` list just to get values.